sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -658,6 +659,23 @@ def test_radius_neighbors_regressor(n_samples=40,
             y_pred = neigh.predict(X[:n_test_pts] + epsilon)
             assert_true(np.all(abs(y_pred - y_target) < radius / 2))
 
+    # test that nan is returned when no nearby observations
+    for weights in ['uniform', 'distance']:
+        neigh = neighbors.RadiusNeighborsRegressor(radius=radius,
+                                                   weights=weights,
+                                                   algorithm='auto')
+        neigh.fit(X, y)
+        X_test_nan = np.ones((1, n_features))*-1
+        empty_warning_msg = ("One or more samples have no neighbors "
+                             "within specified radius; predicting NaN.")
+        pred = assert_warns_message(UserWarning,
+                                    empty_warning_msg,
+                                    neigh.predict,
+                                    X_test_nan)
+        assert_true(np.all(np.isnan(pred)))
+
+
```

Comment:
@abjer Please only leave two blank lines between functions. This flake8 error will not be detected for some reason but we generally don't want to introduce more flake8 errors.

Reply:
Thanks, I have now recommitted.